From c9ecf0bb12c86d6132a2413be0d9692b379f9364 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Mon, 20 Jan 2014 19:06:03 +0200
Subject: [PATCH 523/987] dpaa_eth: remove typedef for dpa_fq_cbs_t

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Fix checkpatch report by removing typedef for dpa_fq_cbs_t.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
Change-Id: Ia7cb7a7524b01ae05fb0849016cdab46abf55de1
Reviewed-on: http://git.am.freescale.net:8181/8117
Reviewed-by: Ruxandra Ioana Radulescu <ruxandra.radulescu@freescale.com>
(cherry picked from commit 500081b4bd86f5b9d4ae00c1307bd1a14df4d2c4)
Reviewed-on: http://git.am.freescale.net:8181/8466
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth.c         | 2 +-
 drivers/net/ethernet/freescale/dpa/dpaa_eth.h         | 4 ++--
 drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c  | 2 +-
 drivers/net/ethernet/freescale/dpa/dpaa_eth_common.h  | 2 +-
 drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c | 2 +-
 drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c  | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
index 0aa4a81..2ac070b 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
@@ -529,7 +529,7 @@ static void priv_ern(struct qman_portal	*portal,
 	dev_kfree_skb_any(skb);
 }
 
-static const dpa_fq_cbs_t private_fq_cbs = {
+static const struct dpa_fq_cbs_t private_fq_cbs = {
 	.rx_defq = { .cb = { .dqrr = priv_rx_default_dqrr } },
 	.tx_defq = { .cb = { .dqrr = priv_tx_conf_default_dqrr } },
 	.rx_errq = { .cb = { .dqrr = priv_rx_error_dqrr } },
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth.h b/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
index 4c934ac..d90ff99 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
@@ -255,13 +255,13 @@ struct dpa_fq {
 	enum dpa_fq_type fq_type;
 };
 
-typedef struct dpa_fq_cbs_t {
+struct dpa_fq_cbs_t {
 	struct qman_fq rx_defq;
 	struct qman_fq tx_defq;
 	struct qman_fq rx_errq;
 	struct qman_fq tx_errq;
 	struct qman_fq egress_ern;
-} dpa_fq_cbs_t;
+};
 
 struct fqid_cell {
 	uint32_t start;
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
index d9a2637..a570d09 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
@@ -1061,7 +1061,7 @@ static inline void dpa_setup_egress(const struct dpa_priv_s *priv,
 	}
 }
 
-void dpa_fq_setup(struct dpa_priv_s *priv, const dpa_fq_cbs_t *fq_cbs,
+void dpa_fq_setup(struct dpa_priv_s *priv, const struct dpa_fq_cbs_t *fq_cbs,
 		struct fm_port *tx_port)
 {
 	struct dpa_fq *fq;
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.h b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.h
index abe3f71..e7bc2af 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.h
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.h
@@ -186,7 +186,7 @@ int dpa_fq_probe_mac(struct device *dev, struct list_head *list,
 int dpa_get_channel(void);
 int dpaa_eth_add_channel(void *__arg);
 int dpaa_eth_cgr_init(struct dpa_priv_s *priv);
-void dpa_fq_setup(struct dpa_priv_s *priv, const dpa_fq_cbs_t *fq_cbs,
+void dpa_fq_setup(struct dpa_priv_s *priv, const struct dpa_fq_cbs_t *fq_cbs,
 		struct fm_port *tx_port);
 int dpa_fq_init(struct dpa_fq *dpa_fq, bool td_enable);
 int __cold __attribute__((nonnull))
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
index 2cc23d5..c0482d6 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_macless.c
@@ -74,7 +74,7 @@ module_param(tx_timeout, ushort, S_IRUGO);
 MODULE_PARM_DESC(tx_timeout, "The Tx timeout in ms");
 
 /* reused from the shared driver */
-extern const dpa_fq_cbs_t shared_fq_cbs;
+extern const struct dpa_fq_cbs_t shared_fq_cbs;
 int __hot dpa_shared_tx(struct sk_buff *skb, struct net_device *net_dev);
 
 /* forward declarations */
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
index c0c618e..1afc2f7 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_shared.c
@@ -104,7 +104,7 @@ static const struct net_device_ops dpa_shared_ops = {
 	.ndo_do_ioctl = dpa_ioctl,
 };
 
-const dpa_fq_cbs_t shared_fq_cbs = {
+const struct dpa_fq_cbs_t shared_fq_cbs = {
 	.rx_defq = { .cb = { .dqrr = shared_rx_dqrr } },
 	.tx_defq = { .cb = { .dqrr = shared_tx_default_dqrr } },
 	.rx_errq = { .cb = { .dqrr = shared_rx_dqrr } },
-- 
1.9.1

